src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using Bogus;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Serilog;
+using Serilog.Configuration;
+using Serilog.Core;
+using Serilog.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace Arcus.Observability.Tests.Integration.Serilog
+{
+    public class ApplicationInsightsSinkTests : IntegrationTest
```

Comment:
Can you have a look why they are considered unit tests during build or why they are not ran please?

Reply:
Ah, is bc of the category.

Reply:
Can we move the category to `IntegrationTest` probably not?